.github/plans/账号系统/09-SSO外部服务接入文档.plan.md

```
@@ -0,0 +1,818 @@
+- `t` 是本次投递时间的毫秒时间戳。
+- `rawBody` 是 HTTP 请求收到的原始 body 字符串。
+- `signing_secret` 是当前首位 active secret hash，也就是后台 Secrets 区域展示的首个值。
+- `v1` 是 HMAC digest 的 Base64URL 字符串。
+
+外部服务验签要求：
+
+1. 读取 `X-Sso-Signature`。
+2. 解析 `t` 和 `v1`。
+3. 检查 `t` 与当前时间的偏差，建议允许 ±5 分钟。
+4. 使用所有仍被外部服务接受的原始 `client_secret` 计算 `SHA-256(client_secret).hex()`。
+5. 对每个派生出的 signing secret 计算 `HMAC-SHA256(signing_secret, `${t}.${rawBody}`)`。
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)